madmom/ml/nn/layers.py

```diff
@@ -287,12 +291,15 @@ class LSTMLayer(Layer):
     """
 
     def __init__(self, input_gate, forget_gate, cell, output_gate,
-                 activation_fn=tanh):
+                 activation_fn=tanh, prev_output=None, prev_state=None):
```

Comment:
I thought about this as well, but they are technically not connected to the gates. The ``prev_output`` is the previous output of the unit, ``prev_state`` is the state, which is the cell weighted by the input gate plus the previous state weighted by the forget gate. Thus I thought it would be the easiest to add them to the layer directly.